nuxt.config.ts

```diff
@@ -9,6 +9,11 @@ export default defineNuxtConfig({
     // TODO: 'nuxt-og-image',
     'nuxt-webhook-validators',
   ],
+  $development: {
+    runtimeConfig: {
+      presetRepo: '',
```

Comment:
I'm currently struggling to determine the proper solution for setting this up in the local module. Here's what I've tried so far:

I've configured the `presetRepo` setting in `local-dev` as follows:
```
// Set initial value for presetRepo
// (Create the type regardless of dev or prod mode to prevent type errors)
updateRuntimeConfig({
  presetRepo: '',
})

if (!nuxt.options.dev) {
  return
}

if (process.argv.includes('--ui-only')) {
  ...

  return
}

const resolver = createResolver(import.meta.url)

// Add preset-repo plugin for local development
addServerPlugin(resolver.resolve('../server/plugins/preset-repo.ts'))
```

My initial goal was the when `runtimeConfig` is resolved and `presetRepo` is not empty, then add server plugin `preset-repo.ts`. I implemented this using the Nuxt `ready` hook, but I'm not confident that this is the best approach.

Any feedback or alternative suggestions from @danielroe would be greatly appreciated!